cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceConfig.java

```diff
@@ -19,23 +19,58 @@
 import com.google.gson.annotations.SerializedName;
 
 import java.util.Objects;
+import javax.annotation.Nullable;
 
 /**
  * Represents the configuration of a namespace. This class needs to be GSON serializable.
  */
 public class NamespaceConfig {
 
-  @SerializedName("scheduler.queue.name")
+
+  public static final String SCHEDULER_QUEUE_NAME = "scheduler.queue.name";
+  public static final String ROOT_DIRECTORY = "root.directory";
+  public static final String HBASE_NAMESPACE = "hbase.namespace";
+  public static final String HIVE_DATABASE = "hive.database";
+
+  @SerializedName(SCHEDULER_QUEUE_NAME)
   private final String schedulerQueueName;
```

Comment:
Just to clarify, I have reverted the changes to this. This comment still shows up because this line doesn't need to change for this.